addons/mail/static/src/core/web/messaging_menu.scss

```diff
@@ -1,7 +1,7 @@
 .o-mail-MessagingMenu {
     width: 450px;
     min-height: 50px;
-    max-height: Min(calc(100vh - 140px), 630px);
+    max-height: Min(calc(70vh - 5px), 630px); // 70vh is the max height of the dropdown menu, so that we will not having two scrolling bar on the right
```

Comment:
As discussed together, original `70vh` in `web/dropdown.scss` can be put in a variable, and it can be reused it. This allows to not put a comment, the coupling is evident from reusing the same variable